ioctl/cmd/account/account.go

```diff
@@ -228,30 +274,41 @@ func newAccountByKeyStore(alias, passwordOfKeyStore, keyStorePath string, wallet
 	return newAccountByKey(alias, hex.EncodeToString(ecrypto.FromECDSA(key.PrivateKey)), walletDir)
 }
 
+func newAccountByPem(alias, passwordOfPem, pemFilePath string, walletDir string) (string, error) {
+	prvKey, err := crypto.ReadPrivateKeyFromPem(pemFilePath, passwordOfPem)
+	if err != nil {
+		return "", output.NewError(output.CryptoError, "failed to read private key from pem file", err)
+	}
+
+	return newAccountByKey(alias, prvKey.HexString(), walletDir)
+}
+
 func storeKey(privateKey, walletDir, password string) (string, error) {
 	priKey, err := crypto.HexStringToPrivateKey(privateKey)
 	if err != nil {
 		return "", output.NewError(output.CryptoError, "failed to generate private key from hex string ", err)
 	}
 	defer priKey.Zero()
 
+	addr, err := address.FromBytes(priKey.PublicKey().Hash())
```

Comment:
not necessary?
HexStringToPrivateKey() will return error if the string cannot make a valid private key

Reply:
i see, move from below